yb-voyager/cmd/importSchema.go

```diff
@@ -81,7 +82,18 @@ func importSchema() {
 	}
 	objectList = applySchemaObjectFilterFlags(objectList)
 	log.Infof("List of schema objects to import: %v", objectList)
-	importSchemaInternal(&target, exportDir, objectList, nil)
+
+	skipFn := func(objType, stmt string) bool {
+		stmt = strings.ToUpper(stmt)
+		return strings.HasPrefix(stmt, "ALTER TABLE")
+	}
```

Comment:
Name this function as `isAlterTableStmt`.

    return objType == "SEQUENCE" && Strings.HasPrefix(...)

We need to skip the alter table statements only from the sequences.sql . Your current implementation skips them from all the files. So the extra condition on `objType` is required.